aten/src/ATen/native/quantized/README.md

```diff
@@ -6,25 +6,26 @@ This document serves as an entry point for quantized kernel implementation.
 
 ## Implementing native quantized ops
 
-The new quantized ops are almost always located under the `ATen/native/quantized/cpu` folder.
-For the sake of an example, let us implement an element-wise quantized logical AND operation under `ATen/native/quantized/cpu/qand.cpp`.
+The new quantized ops are almost always located under the `ATen/native/quantized/cpu` folder. For
+the sake of an example, let us implement an element-wise quantized [logical XAND](https://en.wiktionary.org/wiki/XAND)
+operation under `ATen/native/quantized/cpu/qxand.cpp`.
 
 ### Step 0. Implement the quantized function
 
 Before writing the quantized kernel and registering it, let us implement a quantized function.
 That would assist in any further discussion.
-The snippet below shows the implementation of a quantized AND operator, with the support of all implemented quantized types.
+The snippet below shows the implementation of a quantized XAND operator, with the support of all implemented quantized types.
 
 ```c++
-Tensor quantized_and(Tensor qa, Tensor qb) {
+Tensor quantized_xand(Tensor qa, Tensor qb) {
   // Some type checks for qa and qb should be here...
   Tensor qc;
   double scale = qa.q_scale();
   long zero_point qa.q_zero_point();
 
   auto iter = TensorIterator::binary_op(qc, qa, qb);
 
-  AT_DISPATCH_QINT_TYPES(qa.scalar_type(), "quantized_and", [&]() {
+  AT_DISPATCH_QINT_TYPES(qa.scalar_type(), "quantized_xand", [&]() {
     Tensor qc = at::_empty_affine_quantized(
         qa.sizes(), at::device(kCPU).dtype(SCALAR_TYPE), scale, zero_point);
     cpu_kernel(iter, [&](scalar_t a_value, scalar_t b_value) -> scalar_t {
```

Comment:
I didn't see change of implementation from `and` to `xand`, do you need to change the code below as well?

Reply:
I believe the implementation is abstractly summarized as:

```c++
  Tensor quantized_xand(Tensor qa, Tensor qb) {
    // The awesome op implementation...
    return qc;
  }
```